code-matlab/shared/datatypes/tc/tc.m

```diff
@@ -1,84 +1,50 @@
 function tc_out = tc(varargin)
 %% TC TUNING CURVE DATATYPE
+%   The tc dataype is a container for tuning curves typically used in experimental
+%   analysis.
+%
 %   tc_out = TC(varargin) is a constructor for the tc (tuning curve) struct.
 %
-% youkitan 2014-10-27
+%   INPUTS:
+%
+%
+%   OUTPUTS
+%       iv_out: iv struct with fields:
+%          .tc - [NxM] array, where N is the number of cells and M is the number of
+%                tuning variable bins. Each entry is a normalized firing rate.
```

Comment:
why a *normalized* firing rate? if you simply mean spike counts divided by occupancy, that's a *firing rate*. Normalized suggests something like, normalized suich that 1 is the max firing rate over some stimulus dimension, which is not what we want I think.